README.md

```diff
@@ -37,39 +37,39 @@ The QueryBuilder class is a utility to help construct GraphQL mutations from Far
 Example constructing the GraphQL mutation that upserts an application and deployment.
 
 ```ts
-    // The QueryBuilder manages origin for you
-    const qb = new QueryBuilder(ORIGIN);
+// The QueryBuilder manages origin for you
+const qb = new QueryBuilder(ORIGIN);
```

Comment:
I missed this in my PR but lets make this `'example-origin'` or something as the const hasn't been defined yet in this snippet.

Reply:
feel free to push to my branch and merge